torch/csrc/distributed/c10d/reducer.hpp

```diff
@@ -192,10 +184,9 @@ class TORCH_API Reducer {
   void install_futures(c10::List<c10::intrusive_ptr<c10::ivalue::Future>> futs);
 
   // Returns true if we should rebuild buckets, else false. We only rebuild
-  // buckets once after the first iteration and never rebuild them if
-  // find_unused_parameters_.
+  // buckets once after the first iteration.
   inline bool should_rebuild_buckets() const {
-    return (static_graph_ || !find_unused_parameters_) && !has_rebuilt_bucket_;
```

Comment:
Could you please add the PR summary's descriptions as comments here as well? thanks!

Reply:
sounds good!